src/components/marketplace/order_details.tsx

```diff
@@ -280,4 +375,12 @@ class OrderDetails extends React.Component<Props, State> {
     };
 }
 
-export { OrderDetails };
+const mapStateToProps = (state: StoreState): StateProps => {
+    return {
+        state,
```

Comment:
Is a good practice to map the entire state? What do you think @fvictorio @unjapones ?